networks.js

```diff
@@ -0,0 +1,61 @@
+// All supported networks and related contract addresses are defined here.
+//
+// LINK token addresses: https://docs.chain.link/resources/link-token-contracts/
+// LINK/ETH feeds addresses: https://docs.chain.link/data-feeds/price-feeds/addresses
+// Chain IDs: https://chainlist.org/?testnets=true
+
+require("@chainlink/env-enc").config()
+
+const DEFAULT_VERIFICATION_BLOCK_CONFIRMATIONS = 2
+const SHARED_DON_PUBLIC_KEY =
+  "a30264e813edc9927f73e036b7885ee25445b836979cb00ef112bc644bd16de2db866fa74648438b34f52bb196ffa386992e94e0a3dc6913cee52e2e98f1619c"
+
+const npmCommand = process.env.npm_lifecycle_event
+const isTestEnvironment = npmCommand == "test" || npmCommand == "test:unit"
+
+// Set EVM private key (required)
+const PRIVATE_KEY = process.env.PRIVATE_KEY
+if (!isTestEnvironment && !PRIVATE_KEY) {
+  throw Error("Set the PRIVATE_KEY environment variable with your EVM wallet private key")
+}
+
+const networks = {
+  ethereumSepolia: {
+    url: process.env.ETHEREUM_SEPOLIA_RPC_URL || "UNSET",
+    accounts: PRIVATE_KEY !== undefined ? [PRIVATE_KEY] : [],
+    chainId: 11155111,
+    confirmations: DEFAULT_VERIFICATION_BLOCK_CONFIRMATIONS,
+    linkToken: "0x779877A7B0D9E8603169DdbD7836e478b4624789",
+    linkEthPriceFeed: "0x42585eD362B3f1BCa95c640FdFf35Ef899212734",
+    functionsOracleProxy: "0x649a2C205BE7A3d5e99206CEEFF30c794f0E31EC",
+    functionsBillingRegistryProxy: "0x3c79f56407DCB9dc9b852D139a317246f43750Cc",
+    functionsPublicKey: SHARED_DON_PUBLIC_KEY,
+  },
+  polygonMumbai: {
+    url: process.env.POLYGON_MUMBAI_RPC_URL || "UNSET",
+    accounts: PRIVATE_KEY !== undefined ? [PRIVATE_KEY] : [],
+    chainId: 80001,
+    confirmations: DEFAULT_VERIFICATION_BLOCK_CONFIRMATIONS,
+    linkToken: "0x326C977E6efc84E512bB9C30f76E30c160eD06FB",
+    linkEthPriceFeed: "0x12162c3E810393dEC01362aBf156D7ecf6159528",
+    functionsOracleProxy: "0xeA6721aC65BCeD841B8ec3fc5fEdeA6141a0aDE4",
+    functionsBillingRegistryProxy: "0xEe9Bf52E5Ea228404bB54BCFbbDa8c21131b9039",
+    functionsPublicKey: SHARED_DON_PUBLIC_KEY,
+  },
+  avalancheFuji: {
+    url: process.env.AVALANCHE_FUJI_RPC_URL || "UNSET",
+    accounts: PRIVATE_KEY !== undefined ? [PRIVATE_KEY] : [],
+    chainId: 43113,
+    confirmations: 2 * DEFAULT_VERIFICATION_BLOCK_CONFIRMATIONS,
+    linkToken: "0x0b9d5D9136855f6FEc3c0993feE6E9CE8a297846",
+    linkEthPriceFeed: "0xf4060f80f295b34e0C2471461ba43745Aeb186d6",
```

Comment:
This should be AVAX/LINK, not the ETH/LINK feed.

Also applies to the actual contract's configuration

Reply:
Good point! I will address this in the next PR.